src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxFactoryTests.cs

```diff
@@ -534,6 +534,18 @@ public void TestSpacingOnTernary()
             Assert.Equal("x is DateTime ? y : z", syntaxNode2.ToFullString());
         }
 
+        [Fact]
+        [WorkItem(50742, "https://github.com/dotnet/roslyn/issues/50742")]
+        public void TestLineBreakInterpolations()
+        {
+            var code = @"$""Printed: { new Printer() { TextToPrint = ""Hello world!"" }.PrintedText }""";
+
+            var syntaxNode = SyntaxFactory.ParseExpression(code).NormalizeWhitespace();
+            var aText = syntaxNode.ToFullString();
+
+            Assert.True(!aText.Contains('\n'));
```

Comment:
1. can we have a test with a verbatim interpolated strings as well?
2. i wouldn't just assert the \n bit.  i would also validate the final full string.

Reply:
Yes, it was just laziness 👍.
And for that it would be really great if your _raw-strings_ would exist.

Reply:
Would you add some test? I'm even not sure if the test location is right - but it looked not that bad.

Reply:
In verbatim interpolated strings line breaks in interpolations are okay - so a test is not required in my eyes.
As you know I would prefer if line breaks would be allowed in non verbatim interpolated string. But for now this would help, too.
So, perhaps we should check if a small class compiles without errors after applying NormalizeWhitespace() - It could be formatted in different ways.

Reply:
theyshoudl go in SyntaxNormalizerTests.

Reply:
> In verbatim interpolated strings line breaks in interpolations are okay - so a test is not required in my eyes.

Please add a test.  It will just validate that we do add newlines for verbatim interpolated stirngs.

Reply:
I added a test - it should be fine now. Can you review it again?